src/sdk/Audio/AudioOutputStream.ts

```diff
@@ -61,22 +69,21 @@ export abstract class PullAudioOutputStream extends AudioOutputStream {
      * @member PullAudioOutputStream.create
      * @function
      * @public
-     * @param {AudioStreamFormat} format - The audio data format in which audio will be written to the
-     *        push audio stream's write() method (currently only support 16 kHz 16bit mono PCM).
-     * @returns {PullAudioOutputStream} The push audio input stream being created.
+     * @returns {PullAudioOutputStream} The push audio output stream being created.
      */
-    public static create(format?: AudioStreamFormat): PullAudioOutputStream {
-        return new PullAudioOutputStreamImpl(bufferSize, format);
+    public static create(): PullAudioOutputStream {
+        return new PullAudioOutputStreamImpl();
     }
 
     /**
      * Reads audio data from the internal buffer.
      * @member PullAudioOutputStream.prototype.read
      * @function
      * @public
-     * @returns {Promise<ArrayBuffer>} Audio buffer data.
+     * @param {ArrayBuffer} dataBuffer - An ArrayBuffer to store the read data.
+     * @returns {Promise<number>} Audio buffer length has been read.
      */
-    public abstract read(): Promise<ArrayBuffer>;
+    public abstract read(dataBuffer: ArrayBuffer): Promise<number>;
 
```

Comment:
Is this a breaking change for existing users?

Reply:
Seems it's a breaking change, but I think this change is necessary. Can we have some workaround?